### ISSUE TITLE
[Bug]: (visual) the chip in the CD drive is missing an animation

### ISSUE THREAD

**Issue by aus2004** [FuryFight3r/ScrappingSimulator#68]
### Describe the bug

no animation plays when this specific chip is taken out.

### Screenshot(s)/Video(s)

https://user-images.githubusercontent.com/101810723/189448645-7416cc4d-3432-4793-92bf-73e4699e3d38.mp4

**Comment by FuryFight3r** [comment]
Hi there Aus,
Thank you for reporting this, this has been a bit of an ongoing issue for a while now, due to making changes to prefab components, some parent objects contain components that have been broken from their prefab to be modified separately from the standard component and made unique however, by breaking the prefab connection, when making changes the the base component the changes do not translate to these modified variant components.

I am slowly getting on-top of the missing animations and other issues caused by this, the only issue with fixing these is it breaks the Disassembly of the object as when you load your save into the fixed version, the object will now contain both the old component that has no animation and the new fixed component hidden inside the old one, this new fixed object basically doesn't exist on this loaded objects component list therefore it cannot be removed, thus blocking you from removing the original component that the parent object recognises, in turn being left with an object that can only be sold to the Pawn Shop and cannot be fully disassembled.

I have added a way to fix these broken objects by replacing the bugged object with a completely new version, though for players collecting 'Full' Servers and PCs for when the Achievements comes out would be devastated to find their once Full PC/Server has now been replaced and re-randomised.
But no problem this should be an easy fix, I just need to be mindful of the replacement script and probably modify it to ignore Full PC and Servers, or just Ignore them entirely if they do not require updating 🙂
